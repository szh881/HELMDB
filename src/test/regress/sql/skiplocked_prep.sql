
create table IF NOT EXISTS skiplocked_t1(
    id int,
    info text,
    primary key (id)
);

insert into skiplocked_t1 values (1, 'one');
insert into skiplocked_t1 values (2, 'two');
insert into skiplocked_t1 values (3, 'three');
insert into skiplocked_t1 values (4, 'four');
insert into skiplocked_t1 values (5, 'five');

create table IF NOT EXISTS skiplocked_t2(
    id int,
    info text,
    primary key (id)
)with (STORAGE_TYPE=USTORE);
insert into skiplocked_t2 values (1, 'one');

create table IF NOT EXISTS skiplocked_t3(
    id int,
    info text,
    primary key (id)
)with (ORIENTATION=COLUMN);
insert into skiplocked_t3 values (1, 'one');