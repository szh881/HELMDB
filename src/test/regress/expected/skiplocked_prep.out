create table IF NOT EXISTS skiplocked_t1(
    id int,
    info text,
    primary key (id)
);
NOTICE:  CREATE TABLE / PRIMARY KEY will create implicit index "skiplocked_t1_pkey" for table "skiplocked_t1"
insert into skiplocked_t1 values (1, 'one');
insert into skiplocked_t1 values (2, 'two');
insert into skiplocked_t1 values (3, 'three');
insert into skiplocked_t1 values (4, 'four');
insert into skiplocked_t1 values (5, 'five');
create table IF NOT EXISTS skiplocked_t2(
    id int,
    info text,
    primary key (id)
)with (STORAGE_TYPE=USTORE);
NOTICE:  CREATE TABLE / PRIMARY KEY will create implicit index "skiplocked_t2_pkey" for table "skiplocked_t2"
insert into skiplocked_t2 values (1, 'one');
create table IF NOT EXISTS skiplocked_t3(
    id int,
    info text,
    primary key (id)
)with (ORIENTATION=COLUMN);
NOTICE:  CREATE TABLE / PRIMARY KEY will create implicit index "skiplocked_t3_pkey" for table "skiplocked_t3"
insert into skiplocked_t3 values (1, 'one');
